core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -198,7 +199,7 @@ fun Route.organizations() = route("organizations") {
                 val pagingOptions = call.pagingOptions(SortProperty("name", SortDirection.ASCENDING))
 
                 val infrastructureServicesForOrganization =
-                    infrastructureServiceService.listForOrganization(orgId, pagingOptions.mapToModel())
+                    infrastructureServiceService.listForHierarchy(OrganizationId(orgId), pagingOptions.mapToModel())
```

Comment:
In 2bfccf335d855f8080b5592198fd58a1cd28b486 and related, I renamed `listForOrganization()` (and similar) to `listForId()`. I propose to use the same naming here, for alignment and as it's shorter.

We could later also consider to rename all `...ForId()` functions to just `...For()`, as together with the first argument it anyway reads like "list for organization ID".